modules/lib_ic/doc/src/getting_started.rst

```diff
@@ -3,12 +3,6 @@ Getting Started
 
 Overview
 --------
-
-``lib_ic`` is a library which provides functions that together perform Interference Cancellation (IC)
-on two channel input mic data by adapting to and modelling the room transfer characteristics. ``lib_ic`` library functions
-make use of functionality provided in ``lib_aec`` for the core normalised LMS blocks which in turn uses
-``lib_xcore_math`` to perform DSP low-level optimised operations. For more details refer toc :ref:`ic_overview`.
-
 Repository Structure
```

Comment:
To match other Getting Started files, remove all lines before this one and replace all heading formatting marks with multiple `*`.

In addition, should this file retain the API section?  Some Getting Started files have one and other don't.  Does it contain information not shown elsewhere, e.g. in the API section in the index.rst file?